### PR TITLE
[WIP] ODS-518 test

### PR DESCRIPTION
**Pull request by gmfrasca**
Robot Framework test for ODS-518: Verify the emails provided during installation of RHODS is properly set in the alertmanager and operator Secret

WIP Items:
- Operator installation is through the add-ons tab, not OperatorHub as most other tests. this introduces some unique problems:
  - Test requires Red Hat Account with access to the cluster management page (cloud.redhat.com/openshift)
  - Test requires the Cluster's management URL to be specified in the test-variables.yaml file (not sure if Jenkins can provide this)
  - Test requires the tester to be logged in to the cluster via oc on console (and has admin access)

### CODE REVIEW

File: test-variables.yml.example [hunk 1]
```diff
@@ -3,6 +3,7 @@ BROWSER:
   # List of Chrome options - https://peter.sh/experiments/chromium-command-line-switches/
   # --disable-dev-shm-usage  and --no-sandbox are required for running chromedriver in a container
   OPTIONS: add_argument("--ignore-certificate-errors");add_argument("window-size=1920,1024");add_argument("--disable-dev-shm-usage");add_argument("--no-sandbox")
+OSD_CLUSTERMGMT_URL: "https://console.redhat.com/openshift/details/s/SoMeClUsTeRiDhAsH"
```
Comment (bdattoma) [inline test-variables.yml.example:6]: to add a key-value in the test-variables you may need to apply a similar change to utils/testconfig/generateTestConfigFile and utils/testconfig/test-variables.yml.

**Comment by bdattoma** [comment]
@gmfrasca what's the status on this PR?

**Comment by aloganat** [comment]
Can one of the admins verify this patch?

**Comment by bdattoma** [comment]
Closing this old PR. In case it is still required, please rebase the branch and re-open it, or start a new one

**Pull request closed without merging**